.github/workflows/maven-release.yml

```diff
@@ -2,35 +2,32 @@ name: Publish Release
 on:
   workflow_dispatch:
     inputs:
-      release_core:
-        required: false
-        type: boolean
-        default: false
-        description: Release Core
-
-      release_identity:
-        required: false
-        type: boolean
-        default: false
-        description: Release Identity
-
-      release_lifecycle:
-        required: false
-        type: boolean
-        default: false
-        description: Release Lifecycle
-
-      release_signal:
-        required: false
-        type: boolean
-        default: false
-        description: Release Signal
+      extension:
+        type: choice
+        description: Extension to release
+        options: 
+        - core
+        - identity
+        - lifecycle
+        - signal
+      
+      tag:
+        description: 'tag/version'
+        required: true        
+  
+      action_tag:
+        description: 'Create tag? ("no" to skip)'
+        required: true
+        default: 'yes'
```

Comment:
How does the `boolean` input look like on the Workflow UI? Wondering if we can replace this with a boolean input if feasible.